### PR TITLE
added TBC support

### PR DESCRIPTION
**Pull request by Harapuia**
Added TBC support and added to the tests accordingly. 
tukui tests appear to fail but i'll chalk that up to the site being overloaded. I made no changes to the relevant code, and it shouldnt be affected in the first place

new config setting for it is "tbc"

### CODE REVIEW

File: config.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [WOW ADDON UPDATER]
-WoW Addon Location = C:\Program Files (x86)\World of Warcraft\_retail_\Interface\AddOns
+WoW Addon Location = s:\World of Warcraft\_classic_\Interface\AddOns\
```
Comment (grrttedwards) [inline config.ini:2]: This file’s changes needs to be reverted

**Review by grrttedwards — CHANGES_REQUESTED** [review]
Looks good to me after that one change requested. If tukui tests are failing feel free to add the ignore annotation to them

**Comment by Harapuia** [comment]
There you go. Noticed i swapped retail to tbc too so the entire file is reverted now.

**Comment by Ghosthree3** [comment]
Maybe do edit the comment above `Game Version` to include tbc, `# retail, classic, or tbc`. It's fairly obvious but might help someone and stop bug reports about `bc` not working.

**Comment by Harapuia** [comment]
You're right, easily fixed.
I also noticed the tukui tests are passing now, so it really was a case of the website being overloaded

**Review by grrttedwards — APPROVED** [review]

**Pull request merged**